src/BundleFieldDefinition.php

```diff
@@ -2,11 +2,15 @@
 
 namespace Drupal\commerce;
 
+@trigger_error('The '. __NAMESPACE__ . '\BundleFieldDefinition is deprecated. Instead, use \Drupal\entity\BundleFieldDefinition', E_USER_DEPRECATED);
```

Comment:
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 5 | ERROR | [x] Concat operator must be surrounded by a single space
   |       |     (Squiz.Strings.ConcatenationSpacing.PaddingFound)

Reply:
Fixed. Also figured we should do the same with `BundlePluginInterface`.